src/to_curl.rs

```diff
@@ -385,118 +419,91 @@ mod tests {
     #[test]
     fn examples() {
         let expected = vec![
-            ("xh httpbin.org/get", "curl 'http://httpbin.org/get'", "curl http://httpbin.org/get"),
+            ("xh httpbin.org/get", "curl http://httpbin.org/get"),
             (
                 "xh httpbin.org/post x=3",
-                r#"curl 'http://httpbin.org/post' -H 'content-type: application/json' -H 'accept: application/json, */*;q=0.5' -d '{"x":"3"}'"#,
-                r#"curl http://httpbin.org/post -H "content-type: application/json" -H "accept: application/json, */*;q=0.5" -d "{\"x\":\"3\"}""#,
+                r#"curl http://httpbin.org/post -H 'content-type: application/json' -H 'accept: application/json, */*;q=0.5' -d '{"x":"3"}'"#,
```

Comment:
Powershell seems to strip double quotes unless they are escaped
```
PS C:\> curl http://httpbin.org/post -H 'content-type: application/json' -H 'accept: application/json, */*;q=0.5' -d '{"x":"3"}'
{
  "args": {},
  "data": "{x:3}",
  "files": {},
  "form": {},
  "headers": {
    "Accept": "application/json, */*;q=0.5",
    "Content-Length": "5",
    "Content-Type": "application/json",
    "Host": "httpbin.org",
    "User-Agent": "curl/7.55.1",
    "X-Amzn-Trace-Id": "Root=1-61a25bb0-075508f07946ad341dc7e3f1"
  },
  "json": null,
  "origin": "85.76.18.84",
  "url": "http://httpbin.org/post"
}
PS C:\> curl http://httpbin.org/post -H "content-type: application/json" -H "accept: application/json, */*;q=0.5" -d '{\"x\":\"3\"}'
{
  "args": {},
  "data": "{\"x\":\"3\"}",
  "files": {},
  "form": {},
  "headers": {
    "Accept": "application/json, */*;q=0.5",
    "Content-Length": "9",
    "Content-Type": "application/json",
    "Host": "httpbin.org",
    "User-Agent": "curl/7.55.1",
    "X-Amzn-Trace-Id": "Root=1-61a25bbb-7bdf444d318dfa624eb9bf3a"
  },
  "json": {
    "x": "3"
  },
  "origin": "85.76.18.84",
  "url": "http://httpbin.org/post"
}
```

Reply:
Ah, that's true. It has to be fixed in `os_display`. I didn't realize that shell-escape did handle that properly, but that makes sense because it's inherited from cmd.exe.

I'll put this PR on hold for now.

Reply:
(The reason this is tricky is that PowerShell removes the quotes when spawning an external command, not when parsing the string. And it doesn't remove them when you pass it to an internal commandlet.)

Reply:
Since the last version of xh has this quoting issue, what if we ignore it in this PR and track it in a GitHub issue instead?

Reply:
Does it? Did shell-escape change?

I already wrote a fix FWIW, I just have to get around to deciding whether I like the design and committing/publishing it.

Reply:
It didn't change but we never made use of the cmd.exe-style formatting

```
let escape = if f.alternate() {
    // If formatted with `{:#}`, use cmd.exe-style formatting
    // This is currently not exposed
    shell_escape::windows::escape
}
```

Reply:
Ah, I had already forgotten!

I'll just try to get `os_display` updated this weekend, but that would have been reasonable.